src/components/app/swap/AppSwap.css

```diff
@@ -0,0 +1,18 @@
+.app-swap-container {
```

Comment:
Where's the usecase of `app-swap-container` class?
As I mentioned, pls remove writing custom CSS  and use tailwind CSS instead.

Reply:
this container was already removed, check carefully the code please.

Reply:
What i meant is to remove css contents then.